### DISCUSSION TITLE
Weapon to tag enemies with glowing stains

### DISCUSSION THREAD

**Discussion by Daniel-Chin** [FakeFishGames/Barotrauma#15075] [Ideas and suggestions]
A new small-arm weapon that tags enemies with glowing stains, making them visible in the dark. 

## about alternatives
This is an alternative to  
- thermal goggles
- flashlight
- flare

I'd say this glowy ammunition should come earlier in the tech tree than thermal goggles.

## motivation
Last night I was doing a cave mission, fighting some crawlers in pitch darkness with the following equipment:
- underwater scooter
- harpoon gun (with flashlight)

I wanted to dodge the crawler's charge and then aim. In order to dodge, I had to scoot tangentially w.r.t. the crawler, which prevented the scooter light from revealing the crawler --- I can't dodge something I can't see. 

So, in the current game, the dominant strategy seems to be: don't dodge, just aim and fire, exchanging hp. 

What if you could make your enemy glow in the dark? Will that spice things up?

**Comment by jigardi** [comment]
I like this idea a lot.

Recently my crew and I had a mineral mission to complete deep in a cave. There was no mention of monsters, but we've learned to always assume the worst xD

Our plan was simple:

1. Park the sub such that we could get in/out without moving the sub, and also have a turret perfectly cover us should we need to retreat immediately.
2. Go in with a minimal crew, and our fastest underwater gear; for best odds in case of emergency retreat.

About 5-10 seconds after we left turret range, we were insta-ambushed by elite Tiger-Threshers. There was basically nothing we could do.

The engineer got completely devoured, almost immediately.  I used all my ammo/harpoons in the hopes of saving the engineer, but it didn't seem to affect the Threshers at all.

As I was consigning myself to death, the Tiger-Threshers suddenly both went for the engineer.   I instantly jumped on the opportunity and scurried away on a water-scooter,  towards the light of the turret.

Alas, the elite Tiger-Threshers were too fast, and caught up to me before the guy manning the turret could save me.
With my dying breath I cursed the Tiger-Threshers, and commanded the guy on the turret to blindfire in my general direction.

I died before the first bullets hit, but at least I got my revenge from beyond the grave! xD

In this specific case,  your suggestion could not have saved us - but the same story has happened multiple times on a smaller scale with crawlers and mudraptors, and in those cases it could definitely make a difference!

I'm currently modding a handheld flare gun to try and remedy this,  and I think I can use what I have learned to make your suggestion possible.
